project.clj

```diff
@@ -32,7 +32,8 @@
                  [org.immutant/web "2.1.5"
                   :exclusions [commons-codec]]
                  [riemann-clojure-client "0.4.2"]
-                 [com.novemberain/langohr "3.6.1"]]
+                 [com.novemberain/langohr "3.6.1"]
+                 [prismatic/schema "1.1.2"]]
```

Comment:
I suppose we could, but I'll point out that wasn't released yet at the time this was submitted.